shared/util/feature-flags.native.tsx

```diff
@@ -21,6 +21,7 @@ const ff: FeatureFlags = {
   outOfDateBanner: false,
   plansEnabled: false,
   proofProviders: true,
+  resetPipeline: true,
```

Comment:
We probably either want to leave these off or enable it by default in the service